RePoE/parser/modules/flavour.py

```diff
@@ -0,0 +1,23 @@
+from RePoE.parser.util import write_json, call_with_default_args
+from RePoE.parser import Parser_Module
+
+
+class flavour(Parser_Module):
+    @staticmethod
+    def write(ggpk, data_path, relational_reader, translation_file_cache, ot_file_cache):
+        root = {}
+        for flavour in relational_reader["FlavourText.dat"]:
+            obj = {
+                "text": flavour["Text"],
+                "num": flavour["Unknown0"]
```

Comment:
Since we don't know what this field means, including it doesn't add any value. Could then also go with simply the text as values (`root[flavour["Id"]] = flavour["Text"]`.

Reply:
Good call, I made that change and pushed it.